scripts/bin/GenerateStaticData.pm

```diff
@@ -705,6 +705,7 @@ sub _texts {
         push @texts, localize('Adjusted High Barrier');
         push @texts, localize('Adjusted Low Barrier');
         push @texts, localize('Corporate Action');
+        push @texts, localize('The final settlement price may differ from the indicative price.');
```

Comment:
@ashkanx it's done here, see ^. github keeps the old changes in old conversations now.